src/Navigation/Navigation.Xamarin.Forms/Service/NavigationService.show.cs

```diff
@@ -251,6 +237,37 @@ internal async Task<TViewModelInterface> InitializeViewModelInternal<TViewModelI
             return (TPage)view;
         }
 
+
+        private TPage AddOrUpdateContentCache<TPage>(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this method to have parameters matching all the type parameters.](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5575746)